en-GB/lang/numere.nlng

```diff
@@ -283,6 +284,7 @@ PARSERFUNCS_LISTFUNC_FUNC_SOLVE_[MAT]=						solve(mEqSys)
 PARSERFUNCS_LISTFUNC_FUNC_SPHERE_[DRAW]=					sphere(P,r,STYLE)                                           {}       - Draws a sphere with the radius r at the coordinates P. The color may be modified using STYLE. The plot option light makes the 3d effect visible.
 PARSERFUNCS_LISTFUNC_FUNC_SPLIT_[STRING]=					split(sToSplit,cSep,keepEmpty=false)                       {STR}     - Splits the string sToSplit at the characters cSep in separate strings. With keepEmpty = true empty strings are returned as well, otherwise they are dropped
 PARSERFUNCS_LISTFUNC_FUNC_SQRT_[NUM]=						sqrt(x)                                                     VAL      - Square root of x. %%DEFINED_FOR_ALL%%
+PARSERFUNCS_LISTFUNC_FUNC_STARTSWITH_[STRING]=				startswith(sStr,sStr)                                       STR      - Gives the value True back, when the second Substring is found at the start of the first string. Otherwise, False.

```

Comment:
```suggestion
PARSERFUNCS_LISTFUNC_FUNC_STARTSWITH_[STRING]=				startswith(sStr,sStart)                                     LOG      - Returns true, if the string sStr starts with sStart. Otherwise, false.
```